app/services/image_blob_service.rb

```diff
@@ -3,11 +3,12 @@
 require "mini_magick"
 
 class ImageBlobService < ApplicationService
-  def initialize(temp_image:, proposed_filename:, existing_filenames: [], user: nil)
+  def initialize(temp_image:, proposed_filename:, existing_filenames: [], user: nil, parameterize_name: true)
```

Comment:
TODO make a separate service for the filename logic. Silly for it to be all in this service.